.gitignore

```diff
@@ -4,7 +4,7 @@
 /public/storage
 
 /storage/*.key
-/storage/framework/larave-excel/*
+/storage/framework/larave-excel
```

Comment:
Spelling mistake

Reply:
Good catch! Oh that's why it wasn't ignoring the folder :)

Reply:
Fixed!